### PR TITLE
Add a11y specific markup

### PR DESCRIPTION
**Pull request by dancgray**
Tab order on popup is not correct, should be Title, feedbackMessage then close button.

Also unable to read out feedbackMessage without using arrow keys.

### CODE REVIEW

**Comment by oliverfoster** [comment]
First impression is good. Need to move

File: js/adapt-contrib-tutor.js [hunk 1]
```diff
@@ -101,7 +101,7 @@ define(function(require) {
     });
 
     Adapt.on('questionView:showFeedback', function(view) {
-        Adapt.trigger('popup:opened');
+        Adapt.trigger('popup:opened', this.$el);
         new TutorView({
```
Comment (oliverfoster) [inline js/adapt-contrib-tutor.js]: To pass the tutor view, use:

```
    Adapt.on('questionView:showFeedback', function(view) {
        
        var tutor = new TutorView({
            model: view.model
        });
        
        Adapt.trigger('popup:opened', tutor.$el);

    });
```

File: templates/tutor.hbs [hunk 2]
```diff
@@ -1,21 +1,21 @@
 {{! Maintainers - Kevin Corry, Daryl Hedley}}
-<div class="tutor" role="dialog" aria-label="{{_accessibility._ariaLabels.feedbackPopUp}}">
+<div class="tutor" role="region" aria-label="{{_globals._accessibility._ariaLabels.feedbackPopUp}}">
     <div class="tutor-toolbar">
```
Comment (oliverfoster) [inline templates/tutor.hbs]: Need to move 

```
<div class="tutor-toolbar">
        <a href="#" class="tutor-done" role="button" aria-label="{{_globals._accessibility._ariaLabels.closePopup}}">
            <span class="tutor-icon-close icon icon-cross"></span>
         </a>
     </div>
```

to the bottom of the DOM order and style up at the top accordingly with CSS.

**Comment by oliverfoster** [comment]
Otherwise it's absolutely perfect.

**Comment by dancgray** [comment]
Thanks @oliverfoster , changes applied.

**Comment by oliverfoster** [comment]
One more change for popup focusing

```
        showTutor: function() {
            this.resizeTutor();
            this.$('.tutor').show();
            this.$('.tutor-shadow').fadeIn('slow', _.bind(function() {
                this.$el.a11y_focus();
            }, this));
        },
```

**Comment by oliverfoster** [comment]
+1 perfect

**Comment by moloko** [comment]
+1

**Comment by taylortom** [comment]
+1

**Pull request merged**